src/utils/qmln.ts

```
@@ -0,0 +1,631 @@
+        );
+        this.packet.set(messageHeader);
+        this.packet.set(byteArray, messageHeader.length);
+        this.packet.set(messageEnding, messageHeader.length + byteArray.length);
+    }
+
+    toString() {
+        return Array.from(this.packet).map(b => b.toString(16).padStart(2, '0')).join(' ').toUpperCase();
+    }
+}
+
+export class Function {
```

Biome (1.9.4)

[error] Do not shadow the global "Function" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)